openssl.cmake

```diff
@@ -92,9 +92,9 @@ if(MSVC)
         openssl
         generation
         COMMAND ${CMAKE_COMMAND} -E echo "Makefile generation"
-        COMMAND "cmd /C ms\\do_ms.bat"
+        COMMAND cmd /C "cd ${WORKING_DIRECTORY} && ms\\do_ms.bat"
         COMMAND ${CMAKE_COMMAND} -E echo "... generation completed"
-        WORKING_DIRECTORY <SOURCE_DIR>
+        WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/src/openssl
```

Comment:
AFAIK this should be the same?

And the `cd ${WORKING_DIRECTORY}` in line 95 not needed.

Reply:
${CMAKE_SOURCE_DIR} and ${CMAKE_BINARY_DIR} is the same here. 
Tested it without the cd ${WORKING_DIRECTORY} command and it works.
CI fails because of -WX flag now.

Reply:
what is `-WX` for?